ui/package.json

```diff
@@ -179,7 +178,7 @@
     "tslint": "^5.9.1",
     "tslint-eslint-rules": "^5.4.0",
     "tslint-microsoft-contrib": "^6.0.0",
-    "typescript": ">=3.2.1 <3.3.0",
+    "typescript": ">=3.4.0 <3.6.0",
```

Comment:
still not high enough for https://precisionmedicineinitiative.atlassian.net/browse/RW-5958 unfortunately